bucoffea/monojet/definitions.py

```diff
@@ -380,12 +380,42 @@ def monojet_regions(cfg):
         'leadak4_id',
         # 'veto_vtag'
     ]
+    # Test out different working point for v tagging
+    # the first one is the traditional one used in 2016
     v_cuts = [
         'leadak8_pt_eta',
         'leadak8_id',
         'leadak8_mass',
         'leadak8_tau21',
     ]
+    v_cuts_inclusive = [
+        'leadak8_pt_eta',
+        'leadak8_id',
+    ]
+    v_cuts_loose = [
+        'leadak8_pt_eta',
+        'leadak8_id',
+        'leadak8_mass',
+        'leadak8_wvsqcd_loose',
+    ]
+    v_cuts_tight = [
+        'leadak8_pt_eta',
+        'leadak8_id',
+        'leadak8_mass',
+        'leadak8_wvsqcd_tight',
+    ]
+    v_cuts_looseMD = [
+        'leadak8_pt_eta',
+        'leadak8_id',
+        'leadak8_mass',
+        'leadak8_wvsqcdmd_loose',
+    ]
+    v_cuts_tightMD = [
+        'leadak8_pt_eta',
+        'leadak8_id',
+        'leadak8_mass',
+        'leadak8_wvsqcdmd_tight',
+    ]
```

Comment:
Can we refactor those as:

```python
v_cuts_loose = v_cuts_inclusive + [ ...]
```

In order to avoid having to repeat the same cuts?